include/zephyr/bluetooth/audio/pbp.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_PBP_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_PBP_
+
+/**
+ * @brief Public Broadcast Profile (PBP)
+ *
+ * @defgroup bt_pbp Public Broadcast Profile (PBP)
+ *
+ * @ingroup bluetooth
+ * @{
+ *
+ * [Experimental] Users should note that the APIs can change
+ * as a part of ongoing development.
+ */
+
+#include <zephyr/sys/util.h>
+
+#include <zephyr/bluetooth/audio/audio.h>
+
+/** PBP Roles */
+enum bt_pbp_role {
+	BT_PBP_ROLE_PBS = BIT(0), /*< Public Broadcast Source */
+	BT_PBP_ROLE_PBK = BIT(1), /*< Public Broadcast Sink */
+	BT_PBP_ROLE_PBA = BIT(2), /*< Public Broadcast Assistant */
+};
+
+/** Broadcast Stream encryption state */
+enum bt_pbp_encryption {
+	BT_PBP_STREAMS_NOT_ENCRYPTED = 0,
+	BT_PBP_STREAMS_ENCRYPTED = 1,
+};
+
+/** Broadcast Stream Quality */
+enum bt_pbp_quality {
+	BT_PBP_AUDIO_CONFIG_NOT_PRESENT = 0,
+	BT_PBP_AUDIO_CONFIG_PRESENT = 1,
+};
+
+/** Public Broadcast Announcement features */
+enum bt_pbp_announcement_features {
+	BT_PBP_ANNOUNCEMENT_FEATURE_ENCRYPTION = BIT(0),
+	BT_PBP_ANNOUNCEMENT_FEATURE_STANDARD_QUALITY = BIT(1),
+	BT_PBP_ANNOUNCEMENT_FEATURE_HIGH_QUALITY = BIT(2),
+};
+
+/**
+ * @brief Sets the received PBP role(s) and Public Broadcast Announcement features.
+ *
+ * @param role		PBP role(s) of the device (one or multiple).
+ * @param pba_params	Public Broadcast Announcement features
+ *
+ */
+void bt_pbp_init_source(enum bt_pbp_role role, uint8_t pba_params);
```

Comment:
If we remove the `role`, then what this does is just setting a single value, which arguably shouldn't affect the function that injects the advertising data. I suggest that we remove this function